hivemq-edge/src/frontend/src/modules/Workspace/components/topics/TopicSampler.tsx

```diff
@@ -0,0 +1,36 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import { usePostTopicSamples } from '@/api/hooks/useTopicOntology/useGetTopicSamples.tsx'
+import { Button, Card, CardBody, CardFooter, CardHeader, Heading, HStack, Text } from '@chakra-ui/react'
+
+interface TopicSamplerProps {
+  topic: string
+}
+
+const TopicSampler: FC<TopicSamplerProps> = ({ topic }) => {
+  const { t } = useTranslation()
+  const topicSampler = usePostTopicSamples()
+
+  return (
+    <Card size="sm">
+      <CardHeader as={HStack} justifyContent="space-between">
+        <Heading size="md">{topic}</Heading>
+      </CardHeader>
+      <CardBody data-testid="sampler-warning">
+        <Text>{t('workspace.topicWheel.topicSampler.title')} </Text>
+        <Text>{t('workspace.topicWheel.topicSampler.description')} </Text>
+      </CardBody>
+      <CardFooter>
+        <Button
+          onClick={() => topicSampler.mutateAsync({ adapter: 'sss', topic: 'ss' })}
```

Comment:
question(non-blocking): Wow that's a lot of `s` characters. Hope it's not an Snake adapter of some sort. Is this expected to have that may `s`'s?

Reply:
I knew there was some placeholder left somewhere